roles/initialize_disks/meta/main.yml

```diff
@@ -0,0 +1,53 @@
+galaxy_info:
+  author: Kaloyan Kotlarski
+  description: your role description
+  company: StorPool Inc.
```

Comment:
This is wrong, it's StorPool AD, and we should fix the rest of this file, too (are we using apache or bsd license for this, for example?)

Reply:
Updated